lib/oxidized/model/ftos.rb

```diff
@@ -11,6 +11,8 @@ class FTOS < Oxidized::Model
     cfg.gsub! /^(snmp-server community).*/, '\\1 <configuration removed>'
     cfg.gsub! /(secret \d* {0,1})\S+(.*)/, '\\1<secret hidden>\\2'
     cfg.gsub! /(password \d+) \S+(.*)/, '\\1 <hash hidden>\\2'
+    cfg.gsub! /(snmp-server.*version \S+) \S+(.*)/, '\\1 <community removed>\\2'
```

Comment:
Since these seem to be stand-alone commands (rather than elements of a command), could this perhaps be prefixed with a start of line match to reduce possibility of a false positive? For example, `/^(snmp-server.*version \S+) \S+(.*)/` and `/^(radius-server.*key \d )\S+/)?

Reply:
Indeed, I've made these changes.